src/MixedModelsSim.jl

```diff
@@ -1,17 +1,21 @@
 module MixedModelsSim
 
-using DataFrames, PooledArrays, Tables, Statistics
+using DataFrames
+using Distributions: Chisq, ccdf
+using MixedModels
+using PooledArrays
```

Comment:
@dmbates I know you prefer PooledArrays, but RCall seems to be sensitive to categorical kinda in DataFrames and I haven't yet figured out whether it's for PooledArrays, CategoricalArrays or something about the levels themselves. Just FYI.